packages/reactstrap-validation-date/package.json

```diff
@@ -36,9 +37,11 @@
   },
   "dependencies": {
     "@availity/icon": "^0.4.2",
-    "dayjs": "^1.8.14",
+    "classnames": "^2.2.6",
+    "moment": "^2.24.0",
     "prop-types": "^15.5.8",
-    "react-date-range": "^0.9.4"
+    "react-dates": "^20.2.5",
+    "react-with-direction": "^1.3.0"
```

Comment:
I see this dependency got added but don't see where it's used anywhere. do we need it?

Reply:
`react-with-direction` is a `peerDependency` for `react-dates`